### DISCUSSION TITLE
VDH save  files with numeric number instead than Title or adress

### DISCUSSION THREAD

**Discussion by genaro4** [aclap-dev/video-downloadhelper#65] [Help and Q&A] [answered]
### Your question

I'm a new VDH user and license owner.
I am using VDH on Chrome windows, I tried in several platforms as vimeo or spanbank.
I tried also with the beta version but it doesn't work. I tried with smart naming but doesn't work.

I tried with firefox and it seems that work.

The format of the files is the following 8450455-480p.mp4 , 8450455.mp4,

### Addon & CoApp version and details

version: 8.1.6.0
beta: false
build date: 2024-01-18
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-ES
license: accepted
key (only first 16 fisrt characters): 00816ea869db4eb0
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/121.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\saner",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by paulrouget** [comment]
Ah. That's the first time I hear that one.

So that's Chrome only?

**Comment by genaro4** [comment, marked as answer]
I think ,it's more or less solved.
After some more test, It seems it's only an issue of chrome with my main account and a conflict with one of the others extensions. 
In other chrome user and in Firefox, it works normally ,
I reinstalled chrome, synchronized the user info reinstalled VDH. It works normally but only downloading through the companion app, when I download through the browser ,VDH change the names to numeric  or "td". It's curious because VDH had been working normally until recently.
